docs/antora.yml

```diff
@@ -1,5 +1,5 @@
 name: quarkus-renarde
-title: Quarkus - Renarde
+title: Quarkus Renarde
```

Comment:
Per convention to get proper order on Antora page. And also because we don't want to convey the message that it is a part of Quarkus.